test/support/api/states.js

```diff
@@ -26,6 +26,7 @@ export const stateInProgress = getStateBySlug('in-progress');
 export const stateDone = getStateBySlug('done');
 export const stateUnableToComplete = getStateBySlug('unable-to-complete');
 export const stateThmgTransferred = getStateBySlug('thmg-transferred');
+export const stateEvernorth = getStateBySlug('evernorth');
```

Comment:
Can we delete this one from the fixtures?

Reply:
we probably can